src/gui/src/main.js

```diff
@@ -12,39 +12,46 @@ import VueDatePicker from '@vuepic/vue-datepicker'
 import '@vuepic/vue-datepicker/dist/main.css'
 
 import * as Sentry from '@sentry/vue'
-
+import { useMainStore } from '@/stores/MainStore'
```

Comment:
Why are those changes necessary?

Reply:
there was a problem feeding the application, and you fixed it for me. The problem was with the login page that needed a reload (tested in "private windows")

Reply:
you might want to check this commit https://github.com/taranis-ai/taranis-ai/pull/203/commits/f93466cda01f45534c2f3840ac63b379ef24b6c8